### PR TITLE
Fixes #389: Adding `cpu_mode` to `ovirt_vm`

### LINKED ISSUE #389: Add cpu_mode setting
## Please describe what you would like to see

Add the `cpu_mode` setting to the `ovirt_vm` resource to enable setting the CPU to passthrough.

### PR DESCRIPTION
**Pull request by janosdebugs**
## Please describe the change you are making

Fixing #389 and adding `cpu_mode` to `ovirt_vm`.

## Are you the owner of the code you are sending in, or do you have permission of the owner?

Yes

## The code will be published under the BSD 3 clause license. Have you read and understood this license?

Yes

### CODE REVIEW

**Comment by ovirt-infra** [comment]
All tests passed

**Comment by ovirt-infra** [comment]
All tests passed

**Comment by ovirt-infra** [comment]
All tests passed

File: ovirt/resource_ovirt_vm_test.go [hunk 1]
```diff
@@ -188,6 +188,72 @@ resource "ovirt_vm" "foo" {
...
+	config := fmt.Sprintf(
+		`
+provider "ovirt" {
+	mock = true
+}
+
+resource "ovirt_vm" "foo" {
+	cluster_id  = "%s"
+	template_id = "%s"
+    name        = "test"
```
Comment (engelmi) [inline ovirt/resource_ovirt_vm_test.go:206]: Minor: These lines are inconsistently indented

**Review by engelmi — APPROVED** [review]

**Pull request merged**